### ISSUE TITLE
Javadoc only shows first sentence of documentation for methods

### ISSUE THREAD

**Issue by GlenKPeterson** [Kotlin/dokka#1407]
**Describe the bug**
I expected that if I put a period in my Javadoc for a method, that anything after the period will still show up in the Method Detail section lower down the page, but it just shows the first sentence in both places (it's correct in the Method Summary section).  Seems to work fine for objects/classes, it's methods/fields that are the problem.

### kotlin source file:
```kotlin
    /**
     * Implement this using the implementations in [StringUtils] as a guide to pretty-print
     * Indented data structures for joyful debugging.
     * If you aren't then using `.toString()` for anything else, implement by calling `indentedStr(0)`.
     */
    fun indentedStr(indent:Int):String
```

### HTML output:
```html
<!-- ============ METHOD DETAIL ========== -->
<section role="region">
    <ul class="blockList">
        <li class="blockList"><a id="method.detail">
            <!--   -->
        </a>
            <h3>Method Detail</h3>

            <a id="indentedStr(Integer)">
                <!--   -->
            </a>
            <ul class="blockListLast">
                <li class="blockList">
                    <h4>indentedStr</h4>
                    <pre class="methodSignature"> abstract <a href=https://docs.oracle.com/javase/8/docs/api/java/lang/String.html>String</a> <a href=IndentedStringable.html#indentedStr(Integer)>indentedStr</a>(<a href=https://docs.oracle.com/javase/8/docs/api/java/lang/Integer.html>Integer</a> indent)</pre>
                    <div class="block">Implement this using the implementations in <a href=StringUtils.html>StringUtils</a> as a guide to pretty-print Indented data structures for joyful debugging.</div>

                    <!-- TODO missing return annotation -->
                </li>
            </ul>

        </li>
    </ul>
</section>
```
It's missing `If you aren't then using...` sentence.

**To Reproduce**
Build this project:
https://github.com/GlenKPeterson/Indented

Run with:
```text
./gradlew clean assemble dokkaJavadoc
```

**Dokka configuration**
Default configuration produces this output.

**Installation**
- Operating system: Linux
- Dokka version: 1.4.0-rc
```text
$ ./gradlew --version

------------------------------------------------------------
Gradle 6.3
------------------------------------------------------------

Build time:   2020-03-24 19:52:07 UTC
Revision:     bacd40b727b0130eeac8855ae3f9fd9a0b207c60

Kotlin:       1.3.70
Groovy:       2.5.10
Ant:          Apache Ant(TM) version 1.10.7 compiled on September 1 2019
JVM:          11.0.8 (Ubuntu 11.0.8+10-post-Ubuntu-0ubuntu118.04.1)
OS:           Linux 4.15.0-112-generic amd64
```

**Comment by GlenKPeterson** [comment]
Same thing happens in maven:
```text
$ mvn --version
Apache Maven 3.6.0
Maven home: /usr/share/maven
Java version: 11.0.8, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.15.0-112-generic", arch: "amd64", family: "unix"
```

**Comment by sellophane** [comment]
Hi, this issue is already fixed in 1.4.0, which will be available soon. If you'd like to check it, you can use this post:
https://github.com/Kotlin/dokka/issues/1425#issuecomment-685328000
